modules/rosa-hcp-aws-privatelink-security-groups.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-aws-privatelink-creating-cluster.adoc
+[id="rosa-hcp-aws-privatelink-security-groups_{context}"]
+:_mod-docs-content-type: PROCEDURE
+= Configuring AWS security groups to access the API
+
+With {hcp-title} private clusters, the AWS PrivateLink endpoint exposed in the customer's VPC has a default security group and is applied limited access to the PrivateLink endpoint to only those resources that exist within the VPC, or resources that present with an IP address associated with the VPC CIDR range. In order to grant access to any entities outside of the VPC, either through VPC peering, transit gateway, etc., you will need to create and attach another security group to the PrivateLink endpoint granting the necessary access.
```

Comment:
I think the first sentence here needs to be broken down into two (or more sentences). It is very long and complex.